src/browser_action/browser_action.js

```diff
@@ -1,9 +1,20 @@
 chrome.extension.sendMessage({action: 'getQueue'}, function(response) {
     queue = $('#queue');
     $.each(response, function(issueId, issue) {
-        number = $('<span class="number">').append(issue.number);
-        title = $('<span class="title">').append(issue.title);
-        link = $('<a>', {href: issue.html_url}).append(number).append(':').append(title);
+        repo = $('<span class="issue-repo">').append(issue.repository.name);
+        repo_owner = $('<span class="issue-repo-owner">').append(issue.repository.owner.login);
+        state = $('<span class="issue-state">').append(issue.state);
+        number = $('<span class="issue-number">').append(issue.number);
+        title = $('<span class="issue-title">').append(issue.title);
+        issue_details = $('<div class="issue-details">').append(repo_owner).append(repo).append(number).append(title);
+
+        // NOTE (@talbs): Cale, I needed a DOM element to show the visual icon status. Also, I need animate.css-based HTML classes (animated infinite pulse) attached to this when tests are running.
+        issue_status_icon = ('<span class="issue-status-icon">');
+        issue_status_value = $('<span class="issue-status-value">').append(state);
+
+        // NOTE (@talbs): Cale, styling is expecting class names for states (state-open = open, state-closed = closed, state-merged = merged, tests-passed = pass, tests-failed = failed, tests-running = running). Also, Text values for PR/Issue state and tests state, while  should change as well (Tests Passed, Tests Failed, Tests Running. Need help with wiring, plz.
```

Comment:
@cpennington, need your help with adding HTML classes for different PR+Issue/testing states here.